cli/src/generate/mod.rs

```diff
@@ -74,7 +74,7 @@ pub fn generate_parser_in_directory(
                 .strip_prefix("tree-sitter-")
                 .or_else(|| Some(dir_name.as_ref()))
             {
-                grammar_files::generate_grammar_files(&repo_path, language_name, false)?;
+                grammar_files::generate_grammar_files(&repo_path, language_name)?;
```

Comment:
Should this be removed instead?

Reply:
it should only generate grammar.js i think, but I'm not sure that makes sense anymore since it'd just use the default template to generate from. we could remove it and have the user create it instead, but it's nice to have a template created

Reply:
It will also generate `package.json`.

Reply:
Ja